### ISSUE TITLE
Flathub questions

### ISSUE THREAD

**Issue by DeltaEpsilon19498** [vladimiry/ElectronMail#690]
https://flathub.org/apps/com.github.vladimiry.ElectronMail says that the flathub is unverified. However, according to the readme, this package is verified by the official maintainer. What is the reason for the discrepancy?

Also, there was a recent update to the flathub. However, the build still seems to be based on the release from November of last year. What exactly was changed? Was a dependency updated or a security issue fixed? Or was there some other reason for the update.

Thank you for your help.

**Comment by vladimiry** [comment]
Not a big deal, but technically this is something to ask in https://github.com/flathub/com.github.vladimiry.ElectronMail.

> https://flathub.org/apps/com.github.vladimiry.ElectronMail says that the flathub is unverified. However, according to the readme, this package is verified by the official maintainer. What is the reason for the discrepancy?

I don't remember if the app had "verified" status before at flathub.org, but according to the https://docs.flathub.org/docs/for-users/verification/ absence of "verified" status looks like a flathub's glitch. I guess we need to ask someone from the flathub org to look into it, like maybe @nedrichards or @bilelmoussaoui who originally helped to push the app live at flathub.org via https://github.com/flathub/flathub/pull/1656.

> Also, there was a recent update to the flathub. However, the build still seems to be based on the release from November of last year. What exactly was changed? Was a dependency updated or a security issue fixed? Or was there some other reason for the update.

No app updates happened, but flatpak environment adjusting/updating only, see https://github.com/flathub/com.github.vladimiry.ElectronMail/compare/76a74154ae7048830b716781ddb261f4e5e4ce38...master.

**Comment by DeltaEpsilon19498** [comment]
Thank you.

The new version of Linux Mint, Mint 22, is hiding unverified flatpaks from the software manager by default. You may want to get it verified.

**Comment by vladimiry** [comment]
I thought it's going to be automatically verified at some point, but it looks like I needed to log in to https://flathub.org and click "verify" button on the app. I've just clicked that button and I see this (plus the "Unverify" button):
```
Your app is currently verified by your login @vladimiry with your account on GitHub
```

But it still says "Unverified" on the https://flathub.org/apps/com.github.vladimiry.ElectronMail page. Maybe it's caching or something, let's see if it gets flipped into "verified" after some time.
